index.html

```diff
@@ -0,0 +1,110 @@
+        <li><a href="#">Home</a></li>
+        <li><a href="#">About</a></li>
+        <li><a href="#">Contact</a></li>
+      </ul>
+    </nav>
+    <h1>Decolonizing Artistic Hiring Practices</h1>
+    <p>
+      A conference for the advancement of inclusive, anti-racist, equal, and
+      diverse hiring processes within the arts sector. For Employers,
+      Businesses, and Organisations seeking to make a difference in their
+      community.
+    </p>
+    <p>
+      120 W North Ave #302, Baltimore, MD 21201 November 20, 2020 - 1:15 pm
+    </p>
+    <h2>
+      Panelists
+    </h2>
+    <img
+      src="https://cdn.glitch.com/cfdb336d-7569-49ae-8522-adb0805e7f54%2FMichael.jpg?v=1607559667627"
+      alt="A man with brown skin wearing a red shirt with his arms crossed and smiling towards the camera"
+    />
+    <p>
+      Michael J. Bobbit
+    </p>
+    <p>
+      Michael J. Bobbitt is the Artistic Director of New Repertory Theatre in
+      Boston, MA. He is an arts leader, director, choreographer, playwright and
+      Anti-racist Arts Advocate.
+    </p>
+    <p>
+      Talk # 1: Building Anti-Racist Organisations
+    </p>
+    <p>
+      Michael J. Bobbitt discusses the problem of racism in the workplace. He
+      shares how organasations can conduct anti-racist, equal, and diverse
+      hiring practices.
+    </p>
+    <img
+      src="https://cdn.glitch.com/cfdb336d-7569-49ae-8522-adb0805e7f54%2FEmily.jpg?v=1607559672392"
+      alt="A caucasian woman with brown hair and leather jacket smiling towards the camera"
+    />
+    <p>
+      Emily Elder Cory
+    </p>
+    <p>
+      Emily works with Single Carrot Theatre as their Executive Director! Single
+      Carrot is a professional theatre company in Baltimore City that creates
+      socially relevant theatre in traditional and non-traditional spaces.
+    </p>
+    <p>
+      Talk # 2: Elevated, Equitable, and Accessible Arts Education
+    </p>
+    <p>
+      Emily shares how companies can execute programs, professional development,
+      resources, and events that further improve equality within the arts and
+      education sector.
+    </p>
+    <img
+      src="https://cdn.glitch.com/cfdb336d-7569-49ae-8522-adb0805e7f54%2FSuzan.jpg?v=1607559676067"
+      alt="A woman with brown skin and black hair and glasses smiling towards the camera"
+    />
+    <p>
+      Suzan E. Jenkins
+    </p>
+    <p>
+      Suzan E. Jenkins is a leader in the non-profit arts and culture sector
+      having served in executive positions at the Rhythm and Blues Foundation,
+      the Smithsonian Institution and the Recording Industry Association of
+      America.
+    </p>
+    <p>
+      Talk # 3: Diversity in Positions of Leadership
+    </p>
+    <p>
+      Suzan E. Jenkins discusses leadership and shares a wealth of experience as
+      a distinguished entrepreneur and woman of color in the non-profit arts and
+      culture sector
+    </p>
+    <p class="Buy Tickets">
```

Comment:
A few things here:
- you don't _need_ to have a `p` element around your `button` -- it can easily stand alone, if you want
- in general, your class names should be lowercase (just like your CSS)
- this ends up being two separate classes: `.Buy` AND `.Tickets`, which I don't think you intend; I usually use a class name like `button-wrapper` here because this element **isn't** the button itself, but is a parent/wrapper element.